test/serving.bash

```diff
@@ -48,11 +48,27 @@ function upstream_knative_serving_e2e_and_conformance_tests {
   image_template="registry.ci.openshift.org/openshift/knative-${KNATIVE_SERVING_VERSION}:knative-serving-test-{{.Name}}"
   OPENSHIFT_TEST_OPTIONS="--kubeconfig $KUBECONFIG --enable-beta --enable-alpha --resolvabledomain"
 
+  if [[ $FULL_MESH == "true" ]]; then
+    subdomain=$(oc get ingresses.config.openshift.io cluster  -o jsonpath="{.spec.domain}")
+    OPENSHIFT_TEST_OPTIONS+=" --https --customdomain=$subdomain"
+
+    # Use x509ignoreCN=0.
+    # This should not be necesssary if we could ceate certs with SAN. However, openssl command in the CI
+    # seems old version and so it does not have "-addext" option to add SAN.
+    export GODEBUG="x509ignoreCN=0"
```

Comment:
We should be able to fix that. Can we update the `openssl` command in https://github.com/openshift-knative/serverless-operator/blob/main/openshift/ci-operator/build-image/Dockerfile?

Reply:
I tried but the base image of `registry.ci.openshift.org/openshift/release:golang-1.15` is CentOS 7 and the latest openssl command was still 1.10.

```
$ docker run build-image rpm -qf `which openssl`
openssl-1.0.2k-21.el7_9.x86_64
```